<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddObjectDialog</class>
 <widget class="QDialog" name="AddObjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>298</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>298</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>298</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add object</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc/resources.qrc">
    <normaloff>:/FilesystemWatcher/eye-1.svg</normaloff>:/FilesystemWatcher/eye-1.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Object:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_object">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Select object to watch...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_addFile">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="rsc/resources.qrc">
         <normaloff>:/FilesystemWatcher/file.svg</normaloff>:/FilesystemWatcher/file.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_addFolder">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="rsc/resources.qrc">
         <normaloff>:/FilesystemWatcher/folder.svg</normaloff>:/FilesystemWatcher/folder.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="m_propertiesGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid gray;
    margin-top: 2ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin:  margin;
    subcontrol-position: top; 
    padding: 0 2px 0 2px;
}</string>
     </property>
     <property name="title">
      <string>Properties to watch</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="m_createProp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_modifyProp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Modification</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_deleteProp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_renameProp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Rename</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_alarmGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid gray;
    margin-top: 2ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin:  margin;
    subcontrol-position: top; 
    padding: 0 2px 0 2px;
}</string>
     </property>
     <property name="title">
      <string>Alarm</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="m_useTrayMessage">
        <property name="text">
         <string>Use a tray icon message</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="m_useKeyboardLights">
          <property name="text">
           <string>Use keyboard lights</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Color:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_lightButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="m_soundAlarm">
          <property name="text">
           <string>Sound alarm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_alarmCombo">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>382</width>
       <height>23</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rsc/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddObjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddObjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
